phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/installations/InstallationsFilter.java

```diff
@@ -0,0 +1,69 @@
+package org.phoenicis.javafx.views.mainwindow.installations;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ListChangeListener;
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.FilteredList;
+
+import java.util.function.BiPredicate;
+import java.util.function.Predicate;
+
+/**
+ * Created by marc on 29.03.17.
+ */
+public class InstallationsFilter<E> {
```

Comment:
Does it even make sense to have this class? It's just a copy of `ShortcutFilter`. I could also rename `ShortcutFilter` to `SimpleFilter` and reuse it.